lib/src/ast_extensions.dart

```diff
@@ -100,8 +100,8 @@ extension ExpressionExtensions on Expression {
   ///     ]..addAll(numbers);
   bool get isCollectionLike {
```

Comment:
This method returned the opposite of what its name says. Negated everything in the body and then negated the one callsite so that the name is now accurate.